charts/x509-certificate-exporter/templates/deployment.yaml

```diff
@@ -77,6 +77,9 @@ spec:
       {{- if .Values.secretsExporter.debugMode }}
         - --debug
       {{- end }}
+      {{- if $.Values.exposeRelativeMetrics }}
+        - --expose-relative-metrics
+      {{- else }}
```

Comment:
should be {{- end }}

Reply:
Oh this isn't good at all, thanks for catching it. Looks like it wasn't tested...
We should add a helm linter here :confused:

Reply:
Fixed in 1.10.1